pages/design-develop/tutorials/images/decision-tree.zh-hans.md

```diff
@@ -0,0 +1,92 @@
+---
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after "#".
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:".
+
+title: "替代决策树"
+title_html: "<code>alt</code> 替代决策树"
+lang: zh-hans  # Change "en" to the translated-language shortcode
+last_updated: 2024-05-13  # Keep the date of the English version
+
+translators: 
+- name: "金凯" # Add one -name: line for every translator
+  link: "https://github.com/JinMokai"
+# - name: "Jan Doe"   # Replace Jan Doe with translator name
+# - name: "Jan Doe"   # Replace Jan Doe with name, or delete this line if not multiple translators
+# contributors:
+# - name: "Jan Doe"   # Replace Jan Doe with contributor name, or delete this line if none
+# - name: "Jan Doe"   # Replace Jan Doe with name, or delete this line if not multiple contributors
+
+github:
+  label: wai-tutorials
+
+resource:
+  ref: /tutorials/images/  # Do not change this
+
+navigation:
+  previous: /tutorials/images/imagemap/  # Do not change this
+  next: /tutorials/images/tips/  # Do not change this
+
+permalink: /tutorials/images/decision-tree/zh-hans # Add the language shortcode to the end, with no slash at end, for example: /link/to/page/fr
+ref: /tutorials/images/decision-tree/ # Do not change this
+
+changelog: /tutorials/changelog/
+
+# In the footer below:
+# Translate the Working Group name. Leave the Working Group acronym in English.
+metafooter: true
+editors:
+  - Eric Eggert: "https://www.w3.org/People/yatil/"
+  - Shadi Abou-Zahra: "https://www.w3.org/People/shadi/"
+update_editors:
+  - Brian Elton
+contributing_participants:
+  - 参阅 <a href="/WAI/tutorials/acknowledgements/">致谢</a>
+support: 由教育和外联工作组开发 (<a href="https://www.w3.org/groups/wg/eowg">EOWG</a>)。 在 <a href="https://www.w3.org/WAI/ACT/">WAI-ACT project</a> 支持下开发, 并由 <strong>欧盟委员会<abbr title="信息社会技术">IST</abbr> 计划共同资助</strong>。
+
+# Read Translations Notes for this resource at https://github.com/w3c/wai-tutorials#readme
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="概述" class="full" %}
+{:/}
+
+这个决策树描述了如何在各种情况下使用`<img>`元素的`alt`属性。对于某些类型的图像，可以采用其他方法，例如使用 CSS 背景图像来制作装饰图像，或使用网络字体来代替文本图像。
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+- **图像是否包含文字?**
+  - {:.yes} **是：**
+    -   **…并且文本也以*真实*文本的形式呈现在周围。**
+      _使用空的`alt`属性。参阅 [装饰图像](/tutorials/images/decorative/)。_
+    -   **…并且文字显示只是为了视觉效果。**
+      _使用空的`alt`属性。参阅 [装饰图像](/tutorials/images/decorative/)。_
+    -   **…并且文字还有一个特殊功能，例如是一个图标**
+      _并且使用`alt`属性能传达图像的功能。参阅 [功能性图片](/tutorials/images/functional/)。_
+    -   **…否则图像中的文字将不存在。** _使用`alt`属性包含图片文本。参阅 [文本图像](/tutorials/images/textual/#styled-text-decorative-effect)。_

```

Comment:
```suggestion
    -   **…并且文本并没有在其他地方有所呈现。** _使用`alt`属性包含图片文本。参阅 [文本图像](/tutorials/images/textual/#styled-text-decorative-effect)。_
```

`and the text in the image is not present otherwise.` this can be a bit confusing to translators, I think it means if the image text is not present in other places in the document, an `alt` attribute is needed in this context. Could you please confirm this? @remibetin